src/utilities/__tests__/schemaPrinter-test.js

```diff
@@ -681,14 +771,31 @@ type __Type {
   ofType: __Type
 }
 
+# An enum describing what kind of type a given \`__Type\` is.
 enum __TypeKind {
+
```

Comment:
This newline is slightly odd, if I was writing this be hand I'd omit the initial newline.